cosmo_tester/framework/test_hosts.py

```diff
@@ -533,19 +610,20 @@ class CloudifyMasterManager(_CloudifyManager):
 
     # The MTU is set to 1450 because we're using a static BOOTPROTO here (as
     # opposed to DHCP), which sets a lower default by default
-    NETWORK_CONFIG_TEMPLATE = """DEVICE="eth{0}"
-BOOTPROTO="static"
-ONBOOT="yes"
-TYPE="Ethernet"
-USERCTL="yes"
-PEERDNS="yes"
-IPV6INIT="no"
-PERSISTENT_DHCLIENT="1"
-IPADDR="{1}"
-NETMASK="255.255.255.128"
-DEFROUTE="no"
-MTU=1450
-"""
+    NETWORK_CONFIG_TEMPLATE = """
```

Comment:
use textwrap.dedent here, if you indent the strings: 
```python
    NETWORK_CONFIG_TEMPLATE = textwrap.dedent("""
        DEVICE=...
    """")
```
so that when this is rendered on the machine, it's rendered without the indents

Reply:
Fixed